drivers/hmis/spec/models/hmis/hud/household_spec.rb

```diff
@@ -17,53 +17,84 @@
 
   include_context 'hmis base setup'
 
-  let!(:e1) { create(:hmis_hud_enrollment, project: p1, client: c1, user: u1, data_source: ds1, household_id: '123', entry_date: Date.current) }
-
-  it 'has the right associations' do
-    hh = Hmis::Hud::Household.first
-    expect(Hmis::Hud::Household.all).to contain_exactly(hh)
-    expect(hh.project).to eq(p1)
-    expect(hh.enrollments).to contain_exactly(e1)
-    expect(hh.clients).to contain_exactly(c1)
-    expect(hh.household_size).to eq(1)
-  end
-
-  describe 'scope tests' do
-    let!(:c2) { create :hmis_hud_client, data_source: ds1, user: u1, first_name: 'Test', last_name: 'User' }
-    let!(:e2) { create :hmis_hud_enrollment, project: p1, client: c2, user: u1, data_source: ds1, household_id: '456', entry_date: Date.yesterday }
+  describe 'basic tests' do
```

Comment:
The tests in the 'basic tests' describe block are unchanged, I just moved them into a block so that the 'sort tests' below could have a fresh slate (no existing households in the db) for clarity.

Reply:
Can you add a regression test that would have failed prior to your change?

I think the setup would be like: Sort A-Z where there is a household with a non-HoH member with name "z" but the HoH member has name "a", and make sure that the household is positioned according to the HoH. That's what I was seeing with the bug, that wasn't caught by tests.
(I tested locally and it is working for me)